pkg/pod/container.go

```diff
@@ -235,6 +235,29 @@ func (builder *ContainerBuilder) WithEnvVar(name, value string) *ContainerBuilde
 	return builder
 }
 
+// WithVolumeMounts configures pod volumes to mount inside the container.
+func (builder *ContainerBuilder) WithVolumeMounts(volMounts []v1.VolumeMount) *ContainerBuilder {
```

Comment:
@trewest @kononovn Wondering if we want to have builders for volumeMounts. 

Still I see many functions that take in a `v1.VolumeMount` or `[]v1.VolumeMount`

Reply:
@yprokule Can we change this to `WithVolumeMount`? If you have multiple mounts you can just chain them together (`WithVolumeMount(<def>).WithVolumeMounts(<def>)`)

Reply:
@trewest updated